gradle-examples/composite-build/build.gradle

```diff
@@ -0,0 +1,79 @@
+buildscript {
+    String thorntailVersion = System.getProperty('thorntailVersion') ?: VERSION_THORNTAIL
+
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        maven {
+            url "https://oss.sonatype.org/content/repositories/snapshots/"
+        }
+    }
+
+    dependencies {
+        classpath "io.thorntail:thorntail-gradle-plugin:$thorntailVersion"
+    }
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+    maven {
+        url "https://oss.sonatype.org/content/repositories/snapshots/"
+    }
+}
+
+String thorntailVersion = System.getProperty('thorntailVersion') ?: VERSION_THORNTAIL
+
+apply plugin: 'war'
+apply plugin: 'thorntail'
+
+dependencies {
+
+    compileOnly 'javax:javaee-api:8.0'
+
+    // Include the "platform project" that we have dependency on.
+    implementation platform('io.thorntail.examples.gradle:thorntail-multi-module-app:0.0.0')
+
```

Comment:
... and this is how I am consuming BOM style publications within my other projects.